clarity/src/vm/analysis/tests/mod.rs

```diff
@@ -19,7 +19,7 @@ use vm::analysis::{mem_type_check, AnalysisDatabase};
 use vm::analysis::{type_check, ContractAnalysis};
 use vm::ast::parse;
 
-mod costs;
+// mod costs;
```

Comment:
:eyes:

Reply:
Moved to `clarity_vm` testing in 9e1f96ad4